contracts/RepaymentController.sol

```diff
@@ -134,10 +156,8 @@ contract RepaymentController is IRepaymentController, InterestCalculator, FeeLoo
     function redeemNote(uint256 loanId, address to) external override {
```

Comment:
The `NoteRedeemed` event probably needs a new `bool` field to tell whether the note has been burned

Reply:
The burning of the lenderNote will emit an event in this case. I didnt want to duplicate the logic in the event as well. Going to get with @sandbochs to see what makes the most sense for him from a FE view

Reply:
nvm, OOO till the 22nd. Going to check with @luckyOdisetti

Reply:
Ah true, I think that burn probably covers being able to differentiate. I'm cool w/ either implementation in that case.